pkg/etcd/etcd.go

```diff
@@ -864,6 +865,54 @@ func (e *ETCD) setLearnerProgress(ctx context.Context, status *learnerProgress)
 	return err
 }
 
+// clearAlarms iterates over the list of alarms, attempting to clear any alarms set
+// on the local non-learner member. It will exit once there are no alarms to clear.
+func (e *ETCD) clearAlarms(ctx context.Context) error {
+	<-e.runtime.AgentReady
+	t := time.NewTicker(manageTickerTime)
+	defer t.Stop()
+
+	for range t.C {
+		ctx, cancel := context.WithTimeout(ctx, testTimeout)
+		defer cancel()
+
+		// Check to see if the local node is the leader. Only the leader should do learner management.
+		if e.client == nil {
+			logrus.Error("Etcd client was nil")
```

Comment:
Does this need to be logged since it'll continue anyway?

Reply:
Yeah, this bit is mostly copypasta'd from elsewhere. I let myself get sucked in to some additional cleanup of the cluster code that will address this.

Reply:
I burned a bunch of time trying to refactor the reconcile code to also use the embedded executor's etcd library code instead of calling directly into etcd-io code to start up an embedded etcd, so that we could also clear alarms on the copy of the data that we use for reconciling... but after a good night's sleep I had second thoughts and will hold on to that for a separate PR.